cmd/dep/testdata/gb/golden.txt

```diff
@@ -0,0 +1,6 @@
+Detected gb manifest file...
+Converting from gb manifest...
+  Trying master (a0196ba) as initial lock for imported dep github.com/sdboyer/deptestdos
```

Comment:
As noted above, we would like to be consistent in how these are logged and have the constraints listed before the locked projects.